src/Containers/Impl/Docker/LocalDockerService.cs

```diff
@@ -149,13 +150,24 @@ public Task<string> RepositoryLogoutAsync(RepositoryCredentials auth, Cancellati
 
             var output = await process.StandardOutput.ReadToEndAsync();
             var error = await process.StandardError.ReadToEndAsync();
-            if (!string.IsNullOrEmpty(error)) {
+            if (!string.IsNullOrEmpty(error)  && !IsSecurityWarning(error)) {
                 _outputLogWriter?.Write(MessageCategory.Error, error);
                 throw new ContainerException(error);
             }
 
+            if (!process.HasExited) {
```

Comment:
How is this possible?

Reply:
I missed removing this code. I had added this while testing the "it leaves behind a window" problem.